firmware/keyboards/ErgoTravel/keymaps/fedack/keymap.cpp

```diff
@@ -0,0 +1,248 @@
+  KC_DEL  ,KC_F1  ,KC_F2  ,KC_F3  ,KC_F4  ,KC_F5  , KC_LCBR,   \
+  KC_LSFT ,KC_F7  ,KC_F8  ,KC_F9  ,KC_F10 ,KC_F11 , KC_SPC,  \
+  KC_LCTL , KC_LGUI, KC_LALT, LAYER_3, LAYER_1, KC_SPC,_______ \
+);
+
+     /* Layer 2 (lower)
+ * ,------------------------------------------------.
+ * |  ~   |   !  |   @  |   #  |   $  |   %  |   _  |
+ * |------+------+------+------+------+-------------|
+ * | Del  |  F1  | F2   |  F3  | F4   |  F5  |  (   |
+ * |------+------+------+------+------+------|------|
+ * | Shift|  F7  | F8   |  F9  |  F10 | F11  |Space |
+ * |------+------+------+------+------+------+------'
+ * | Ctrl | GUI  | Alt  | L(3) | L(1) |Space |
+ * `-----------------------------------------'
+ */
+    uint32_t layer2[MATRIX_ROWS][MATRIX_COLS] =
+        KEYMAP( \
+  KC_TILD ,KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_UNDS,  \
+  KC_DEL  ,KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_LPRN,   \
+  KC_LSFT ,KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_SPC,  \
+  KC_LCTL ,KC_LGUI, KC_LALT, LAYER_3, LAYER_1, KC_SPC,  _______ \
+);
+
+     /* Layer 3
+ * ,------------------------------------------------.
+ * | ESC* |  F1  |  F2  |  F3  |  F4  |  F5  |PRTSCR| 
+ * |------+------+------+------+------+-------------|
+ * | CAPL*| PUP* | NEXT*| PLAY*| PREV*| VOL+*| MUTE*|
+ * |------+------+------+------+------+------|------|
+ * | Shift|PDOWN*| INS* | HOME*| END* | VOL-*|Space |
+ * |------+------+------+------+------+------+------'
+ * | Ctrl | GUI  | Alt  | L(3) | L(1) |Space |
+ * `-----------------------------------------'
+ */
+    uint32_t layer3[MATRIX_ROWS][MATRIX_COLS] =
+        KEYMAP( \
+  KC_ESC,       KC_F1,      KC_F2,              KC_F3,                  KC_F4,                  KC_F5,      KC_PSCREEN,  \
+  KC_CAPSLOCK,  KC_PGUP,    KC_MEDIA_NEXT_TRACK,KC_MEDIA_PLAY_PAUSE,    KC_MEDIA_PREV_TRACK,    KC_VOLU,    KC_MUTE,   \
+  KC_LSFT,      KC_PGDN,    KC_INS,             KC_HOME,                KC_END,                 KC_VOLD,    KC_SPC,  \
+  KC_LCTL ,     KC_LGUI,    KC_LALT,            LAYER_3,                LAYER_1,                KC_SPC,     _______ \
+);
+
+    /*
+     * add the other layers on the regular presses.
+     */
+    for (int row = 0; row < MATRIX_ROWS; ++row)
+    {
+        for (int col = 0; col < MATRIX_COLS; ++col)
+        {
+            matrix[row][col].addActivation(_L1, Method::PRESS, layer1[row][col]);
+            matrix[row][col].addActivation(_L2, Method::PRESS, layer2[row][col]);
+            matrix[row][col].addActivation(_L3, Method::PRESS, layer3[row][col]);
+        }
+    }
+
+    // if you want to add Tap/Hold or Tap/Doubletap activations, then you add them below.
+
+}
+
+#endif  // left
+
+
+
+#if KEYBOARD_SIDE == RIGHT
+
+ /* Qwerty
+ * ,------------------------------------------------.
+ * |   =  |   Y  |   U  |   I  |   O  |   P  | Bksp |
+ * |------+------+------+------+------+-------------|
+ * |   ]* |   H  |   J  |   K  |   L  |   ;  |  "   |
```

Comment:
Actually, the ] doesn't make sense without opening brackets on the other side. Change it to \